### PR TITLE
Bind Event Listener when registering display object

### PR DESCRIPTION
**Pull request by sudhir-jedhe**
inital commit

### CODE REVIEW

File: src/RoleObjectFactory.js [hunk 1]
```diff
@@ -57,6 +57,8 @@ import ToolBarData from './RoleObjects/ToolBarData.js';
  * @param {number} [config.containerIndex] - An optional value for the layer index to add the DisplayObject
  * @param {String} [config.domIdPrefix] - Optional parameter for the prefix to use for the DOM id in the translated display object.  Defaults to 'acc_'
  * @param {Function} [config.onFocus] - Optional callback when the DisplayObject is brought into focus
+ * @param {Function} [config.onBlur] - Optional callback when the DisplayObject is gets blur
+ * @param {Function} [config.onValueChanged] -  Optional callback when the value changed for the DisplayObject
```
Comment (mnewcomb314) [inline src/RoleObjectFactory.js]: CAM currently can generate about a dozen unique events between all the roles (documented in the "Events" part of the readme.md).  So, I would like something more generic that can cover all of them.  I was thinking that to the `config` object there could be an optional `events` object.  The `events` object would have fields named for the event and the value being the event listener.  While I haven't tested this code, it should allow for the event registration code to be along the lines of:
```
if (config.events) {
  _.forEach(config.events, (eventName, listener) => {
    displayObject.on(eventName, listener);
  });
}
```
That does mean the users of CAM can bind event listeners for a role that will never fire that event or an event that CAM will never fire (but those cases are already handled by documenting roles and their events in the readme.md), but it allows for consistent event registration regardless of role.  This also allows for the previous code for onFocus and onKeyboardClick to be removed since those should use the `events` object going forward (since this is part of the 1.0 release, API breaking changes are okay).

File: src/RoleObjects/AccessibilityObject.js [hunk 2]
```diff
@@ -710,4 +710,18 @@ export default class AccessibilityObject {
       evt.preventDefault();
     }
   }
+  /**
+   * Event listener for valueChanged events
+   * @access private
+   * @param {SyntheticEvent} evt - React event
+   */
+  _onValueChanged(evt) {
```
Comment (mnewcomb314) [inline src/RoleObjects/AccessibilityObject.js]: Why is this needed?  This is something I would only a subset of interactive roles to have.

File: src/RoleObjectFactory.js (hunk 1 above)
Comment (sudhir-jedhe) [reply src/RoleObjectFactory.js]: updated

File: src/RoleObjects/AccessibilityObject.js (hunk 2 above)
Comment (sudhir-jedhe) [reply src/RoleObjects/AccessibilityObject.js]: removed

Comment (mnewcomb314) [reply src/RoleObjects/AccessibilityObject.js]: you might need to push your local branch.  I'm still seeing this function in the PR.

**Comment by mnewcomb314** [comment]
Please bump the CAM version in package*.json to `1.0.0-prerelease.3` as prep for a release build once these changes are merged in.

**Comment by mnewcomb314** [comment]
Perfect!
